.travis/mageia/travis-tasks.sh

```diff
@@ -4,7 +4,7 @@
 set -e
 set -x
 
-
+PROCESSORS=$(/usr/bin/getconf _NPROCESSORS_ONLN)
 COMMON_MESON_ARGS="-Dtest_dirty_git=${DIRTY_REPO_CHECK:-true} -Ddeveloper_build=false"
```

Comment:
Related to my comments for `.travis/fedora/travis-tasks.sh`, should `$COMMON_MESON_ARGS` be carried through to each of the `meson test ...` runs? Or should it just be dropped. It appears that for mageia, the dirty git tests are never run, and `developer_build` enables running of the `smoketest` and `formatters` suites--which are explicitly called out below.

Reply:
I dropped the formatter suite but left the smoketest suite (since it still runs the header-import test).

I'm not actually sure why Mageia wasn't running the dirty test. I'm adding the dirty test to the 'ci' suite so it doesn't need a separate entry in the `travis-tasks.sh` files. Hopefully that will force it to work on Mageia too.